tools/packaging/kata-deploy/README.md

```diff
@@ -43,6 +43,34 @@ $ kubectl apply -f kata-rbac/base/kata-rbac.yaml
 $ kubectl apply -k kata-deploy/overlays/rke2
 ```
 
+#### [k0s] cluster
+
+For your [k0s](https://k0sproject.io/) cluster, run:
+
+```sh
+$ git clone https://github.com/kata-containers/kata-containers.git
+```
+
+Check and switch to the stable branch of your choice, if wanted, and then run:
```

Comment:
I'd make a note that, **for now**, this will only work on the `main` branch, and that "stable" is not supported.